.github/workflows/ci.yml

```diff
@@ -310,9 +310,11 @@ jobs:
       - name: Install
         run: |
           cmake --build build --target install --config "${{ matrix.cfg.type }}" -j 2
+          cd build/install
+          zip -y -r ../../plasma-${{ matrix.platform.str }}-${{ matrix.cfg.str }}.zip *
```

Comment:
Can you remind me what the issue with GitHub's zipping was? Something about corrupting app bundles?

Reply:
macOS bundles contain symlinks - which by default zip tools typically turn into copies of the symlinked file/folder. I believe Github's built in zip was having the same issue.